scripts/build_dashboard.py

```
@@ -462,7 +504,9 @@ def generate_html(
   return Object.keys(data[kind]).map(k => ({{x:data.dates,y:data[kind][k],name:k,type:'scatter',mode:'lines'}}));
 }}
 Plotly.newPlot('equity', traces('equity'), {{paper_bgcolor:'rgba(0,0,0,0)',plot_bgcolor:'rgba(0,0,0,0)',font:{{color:getComputedStyle(document.body).getPropertyValue('--text')}}}});
-Plotly.newPlot('dd', traces('drawdown'), {{paper_bgcolor:'rgba(0,0,0,0)',plot_bgcolor:'rgba(0,0,0,0)',font:{{color:getComputedStyle(document.body).getPropertyValue('--text')}},yaxis:{{tickformat:'.0%'}}}});
+Plotly.newPlot('dd', traces('drawdown'), {paper_bgcolor:'rgba(0,0,0,0)',plot_bgcolor:'rgba(0,0,0,0)',font:{color:getComputedStyle(document.body).getPropertyValue('--text')},yaxis:{tickformat:'.0%'}});
```

Ruff (0.15.0)

[error] Undefined name `paper_bgcolor`

(F821)